help/mobile-notifications.md

```diff
@@ -39,6 +35,49 @@ notifications while you are actively using one of the Zulip apps.
 
 {end_tabs}
 
+## Testing mobile notifications
+
+Start by configuring your notifications settings to make it easy to trigger a
+notification.
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{settings_tab|notifications}
+
+1. In the **Mobile** column of the **Notification triggers** table, make sure
+   the **DMs, mentions, and alerts** checkbox is checked.
+
+1. Under **Mobile message notifications**, make sure the **Send mobile
+   notifications even if I'm online** checkbox is checked.
+
+{end_tabs}
+
+Next, test Zulip push notifications on your mobile device.
+
+{start_tabs}
+
+{tab|mobile}
+
+1. [Download](https://zulip.com/apps/) and install the Zulip mobile app if you
+   have not done so already.
+
+1. If you Zulip organization is self-hosted (not at `*.zulipchat.com`),
+   [check](/help/mobile-notifications#enabling-push-notifications-for-self-hosted-servers)
+   whether push notifications have been set up. If they were set up recently,
+   you will need to [log out](/help/logging-out) of your account and [log
+   in](/help/logging-in) again.
+
+1. Navigate to the **home screen** in your device so that the Zulip app is
```

Comment:
Maybe we should add:
```suggestion
1. Close the Zulip app, or navigate to the **home screen** in your device so that the Zulip app is
```

Reply:
@gnprice or @chrisbobbe could you help make sure we write the correct thing here for making sure mobile push notifications will be triggered?

Reply:
Hmm, good question.

I think in fact we can just leave this step out — it doesn't actually help anything. We show notifications just the same regardless of whether the app is running in the foreground, running in the background, or (for the most part) not running.

The one thing to avoid is actually to force the app closed. Even that's probably fine — I can't actually reproduce a problem when I try it now — but reportedly, in general, if you force-quit an app that can cause the OS to decide you don't want to see notifications from it.

Years ago we did suppress notifications on iOS if you had the app running in the foreground: https://github.com/zulip/zulip-mobile/issues/4241 . And I guess in 2017 when the version of these instructions for the server-admin docs was written, we must have done so on Android as well. But we no longer do that, for the reason described at https://github.com/zulip/zulip-mobile/issues/3114#issuecomment-585966226 .

Reply:
(The last commit that made any substantive edits to these testing instructions was https://github.com/zulip/zulip/commit/e1110308fcbfe692b6e80899afac8376a0c43f63 in June 2017, or arguably 7f460569321b4ed266370b02c5a5397c42902bea that August. So we'd clearly forgotten about them and it's well past time we gave them a refresh; thanks for taking care of this!

That latter commit also explains the otherwise puzzling focus that the instructions had on Android.)

Reply:
Thanks for that detailed context!